src/main/java/it/eng/idsa/businesslogic/processor/consumer/ConsumerSendDataToBusinessLogicProcessor.java

```diff
@@ -71,7 +74,7 @@ public void process(Exchange exchange) throws Exception {
 		headesParts.put("Content-Type", contentType);
 		
 		// TODO: Send The MultipartMessage message to the WebaSocket
-		if(isEnabledIdscp) {
+		if(isEnabledIdscp || isEnabledWebSocket) { //TODO Try to remove this config property
```

Comment:
- We shoud analyze this part of the code, depends of the other logic in the application...